### PR TITLE
Add Content Distribution web story

### PR DESCRIPTION
**Pull request by swissspidy**
Makes progress on #1571 

This is the fourth and last web story for the 2020 Web Almanac.

Also includes a typo fix in the Content Publishing story I happened to spot.

I compressed images locally already. No resizing though as there weren't any extremely large images I think.

### CODE REVIEW

**Comment by bazzadp** [comment]
@swissspidy, I was seeing a flash of text during load.

Finally managed to capture it using WebPageTest: https://www.webpagetest.org/video/compare.php?tests=210118_DiTZ_be1c4220d070926fb6f747507b4c4f6d-r%3A1-c%3A0&thumbSize=200&ival=16.67&end=visual

Loading filmstrip

Any ideas?

**Comment by swissspidy** [comment]
> @swissspidy, I was seeing a flash of text during load.
> 
> Finally managed to capture it using WebPageTest: [webpagetest.org/video/compare.php?tests=210118_DiTZ_be1c4220d070926fb6f747507b4c4f6d-r%3A1-c%3A0&thumbSize=200&ival=16.67&end=visual](https://www.webpagetest.org/video/compare.php?tests=210118_DiTZ_be1c4220d070926fb6f747507b4c4f6d-r%3A1-c%3A0&thumbSize=200&ival=16.67&end=visual)
> 
> Loading filmstrip
> 
> Any ideas?

strange, looks like some images are not loading correctly 🤔 

The first page should look like this:

Screenshot 2021-01-18 at 15 22 12

Maybe I got the paths wrong again somewhere, I'll have to check.

**Comment by bazzadp** [comment]
Looks like paths are `p3` but images are have `p4` in their name instead.

**Comment by swissspidy** [comment]
> Looks like paths are `p3` but images are have `p4` in their name instead.

Not sure what you mean? I don't see `p3` referenced anywhere

**Comment by bazzadp** [comment]
> > Looks like paths are `p3` but images are have `p4` in their name instead.
> 
> Not sure what you mean? I don't see `p3` referenced anywhere

Ignore me - wrong file!

File: src/templates/en/2020/stories/content_distribution.html [hunk 1]
```diff
@@ -0,0 +1,1767 @@
...
+                <div class="fill amp-wp-b3b6893" data-amp-original-style="background-color:#545f74"></div>
+              </div>
+            </div>
+            <div data-amp-original-style="position:absolute;pointer-events:none;left:9.95146%;top:79.61165%;width:80.09709%;height:43.85113%;opacity:1" class="amp-wp-e8247a0">
+              <div class="mask amp-wp-a1b70b8" id="el-a2577370-d911-435c-a21d-826228c08231" data-amp-original-style="pointer-events:initial;width:100%;height:100%;display:block;position:absolute;top:0;left:0">
+                <div data-leaf-element="true" data-amp-original-style="position:absolute;width:100%;height:100.08762%;left:0%;top:-0.04381%" class="amp-wp-3fc8e8f">
+                  <amp-img layout="fill" src="/static/stories/web-almanac-2020-content-distribution/WA_Chat-Window.png" alt="A graphic showing an example website" srcset="/static/stories/web-almanac-2020-content-distribution/WA_Chat-Window.png 994w, assets/WA_Chat-Window-768x631.png 768w, assets/WA_Chat-Window-300x247.png 300w, assets/WA_Chat-Window-150x123.png 150w" class="i-amphtml-layout-fill i-amphtml-layout-size-defined" i-amphtml-layout="fill"></amp-img>
```
Comment (bazzadp) [inline src/templates/en/2020/stories/content_distribution.html:619]: We're missing some of the paths in the images - except for first one:

```suggestion
                  <amp-img layout="fill" src="/static/stories/web-almanac-2020-content-distribution/WA_Chat-Window.png" alt="A graphic showing an example website" srcset="/static/stories/web-almanac-2020-content-distribution/WA_Chat-Window.png 994w, /static/stories/web-almanac-2020-content-distribution/WA_Chat-Window-768x631.png 768w, /static/stories/web-almanac-2020-content-distribution/WA_Chat-Window-300x247.png 300w, /static/stories/web-almanac-2020-content-distribution/WA_Chat-Window-150x123.png 150w" class="i-amphtml-layout-fill i-amphtml-layout-size-defined" i-amphtml-layout="fill"></amp-img>
```

More than this example.

Comment (bazzadp) [reply src/templates/en/2020/stories/content_distribution.html]: Fixed it with a quick search and replace and looks a better now!

Comment (swissspidy) [reply src/templates/en/2020/stories/content_distribution.html]: Thanks @bazzadp! Not sure  how my search+replace missed those instances.

**Review by bazzadp — APPROVED** [review]

**Pull request merged**